File: ejercicios/python/ejercicio056.py
def factorial_iterativo(numero):
   factorial = 1
   for i in range(1,numero):
      factorial *= i
   return factorial

if __name__ == "__main__":
   numero = int(input("ingrese el numero : "))
   if (numero >= 0):
      print ("el factorial de " + str(numero) + " es : " + str(factorial_iterativo(numero)))
   else:
      print("el factorial solo se puede calcular a numeros enteros positivos ")
      

File: ejercicios/c++/ejercicio056.cpp
#include <iostream>
using namespace std;

long factorial_iterativo(int numero){	long factorial = 1;	for(int i = 1; i <= numero; i++){		factorial *= i;	}	return factorial;
}

int main() {
	int numero;
	cout << "ingrese el numero : ";
	cin >> numero;
	
	if (numero >= 0)
		cout << "el factorial de " << numero << " es : " << factorial_iterativo(numero) << endl;
	else
		cout << "el factorial solo se puede calcular a numeros enteros positivos " << endl;
	
	return 0;
}



